get-started/06_bind_mounts.md

```diff
@@ -55,7 +55,7 @@ So, let's do it!
     docker run -dp 3000:3000 `
         -w /app -v "$(pwd):/app" `
         node:12-alpine `
-        sh -c "yarn install && yarn run dev"
+        sh -c "apk add --no-cache python g++ make && yarn install && yarn run dev"
```

Comment:
Looks like the bullet at line 65 also needs to be updated (as it explains the command).

Also considering if we should use the image we built in "part 2: sample application" for this, as that would already have these dependencies installed (or if we can update the to not require these dependencies, as it may distract from the main purpose).

Reply:
Installing `python` failed on an arm64 machine. After running the below command:
```
docker run -dp 3000:3000 \
        -w /app -v "$(pwd):/app" \
        node:12-alpine \
        sh -c "apk add --no-cache python g++ make && yarn install && yarn run dev"
```
This error was produced:

```
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/aarch64/APKINDEX.tar.gz
ERROR: unable to select packages:
  python (no such package):
    required by: world[python]
```

```suggestion
        sh -c "apk add --no-cache python2 g++ make && yarn install && yarn run dev"
```